docs/source/tutorials/passes/onnx.md

```diff
@@ -134,7 +134,7 @@ Please refer to [IncQuantization](inc_quantization), [IncDynamicQuantization](in
 [IncStaticQuantization](inc_static_quantization) for more details about the passes and their config parameters.
 
 ### Quantize with AMD Vitis AI Quantizer
-Olive also integrates [AMD Vitis AI Quantizer](https://github.com/microsoft/Olive/blob/main/olive/passes/onnx/vitis_ai/qdq_quantizer.py) for quantization.
+Olive also integrates [AMD Vitis AI Quantizer](https://github.com/microsoft/Olive/blob/main/olive/passes/onnx/vitis_ai/quantizer.py) for quantization.
```

Comment:
I think it would be better to link to https://github.com/microsoft/Olive/blob/main/olive/passes/onnx/vitis_ai/quantize.py here